nmigen_boards/chameleon96.py

```diff
@@ -0,0 +1,120 @@
+import os
+import subprocess
+
+from nmigen import *
+from nmigen.build import *
+from nmigen.vendor.intel import *
+from .resources import *
+
+
+__all__ = ["Chameleon96Platform"]
+
+class Chameleon96Platform(IntelPlatform):
+    device      = "5CSEBA6" # Cyclone V SE 110K LEs
+    package     = "U19"     # UBGA-484
+    speed       = "I7"
+    default_clk = "cyclonev_oscillator"
+    resources   = [
+        # WIFI and BT LEDs
+        *LEDResources(
+            pins="Y19 Y20",
+            attrs=Attrs(io_standard="2.5 V")),
+
+        # TDA19988 HDMI transmitter
+        Resource("tda19988", 0,
+            Subsignal("vpa",       Pins("    W8  W7  V6  V5  U6 ", dir="o")), # bits 3 to 7
+            Subsignal("vpb",       Pins("AB5 AA5 AA8 AB8 AB9 Y11", dir="o")), # bits 2 to 7
+            Subsignal("vpc",       Pins("    W6  Y5  AB7 AA7 AA6", dir="o")), # bits 3 to 7
+
+            Subsignal("pclk",      Pins("AB10", dir="o")),
+            Subsignal("hsync",     Pins("V10 ", dir="o")),
+            Subsignal("vsync",     Pins("V7  ", dir="o")),
+            Subsignal("de",        Pins("Y8  ", dir="o")),
+
+            Subsignal("cscl",      Pins("U7  ", dir="o")),
+            Subsignal("csda",      Pins("U10 ", dir="io")),
+
+            Subsignal("i2s",
+                Subsignal("mclk",  Pins("V11 ", dir="o")), # OSC_IN/AP3
+                Subsignal("txd",   Pins("AA11", dir="o")), # AP1
+                Subsignal("txc",   Pins("W11 ", dir="o")), # ACLK
+                Subsignal("txfs",  Pins("V9  ", dir="o")), # AP0
+            ),
+            Attrs(io_standard="1.8 V")
+        ),
+
+        # Wifi and BT module
+        Resource("wifi", 0,
+            Subsignal("clk",       Pins("AB20", dir="o")),
+            Subsignal("cmd",       Pins("AB18", dir="io")),
+            Subsignal("data",      Pins("Y14 AB19 AA18 AB15", dir="io")),
+            Attrs(io_standard="1.8 V"),
```

Comment:
This is SDIO, right? Is there a reason to not use `SDCardResources`?

Reply:
Thanks for the hint. No, there's no reason, will change it.